src/lib/loaders/loaders_without_authentication/gravity.js

```diff
@@ -98,5 +98,6 @@ export default opts => {
     ),
     tagLoader: gravityLoader(id => `tag/${id}`),
     trendingArtistsLoader: gravityLoader("artists/trending"),
+    usersLoader: gravityLoader("users"),
```

Comment:
Nah, you can get rid of this one, we just want the authenticated one (which only exists in a GraphQL request if you pass along the `X-Access-Token`with your request).